src/components/BackgroundCanvas/BackgroundCanvas.component.tsx

```diff
@@ -277,9 +277,8 @@ const BackgroundCanvas = () => {
   }
 
   const animate = () => {
-    if(bgCanvas && bgCanvas.current && bgCanvas.current.getContext) {
-
-      let canvas = bgCanvas.current as HTMLCanvasElement;
+    if (bgCanvas && bgCanvas.current && typeof bgCanvas.current.getContext === 'function') {
```

Comment:
Instead of changing the code to make the typescript work, lets change the types to make the code work